plugins/main/server/health-check/alerting-monitors.ts

```diff
@@ -0,0 +1,276 @@
+      },
+    },
+    actions: [] as TriggerAction[],
+  };
+  if (destinationId) {
+    triggerBase.actions.push({
+      name: 'Send notification',
+      destination_id: destinationId,
+      message_template: {
+        lang: 'mustache',
+        source: message || 'Sample notification',
+      },
+      subject_template: { lang: 'mustache', source: 'Alerting' },
+    });
+  }
+
+  return {
+    type: 'monitor',
+    name,
+    enabled: false,
+    monitor_type: 'query_level_monitor',
+    schedule: { period: { interval: 1, unit: 'MINUTES' } },
+    // Minimal query-level input that works across installations
+    inputs: [
+      {
+        search: {
+          indices: [WAZUH_ALERTS_PATTERN],
+          query: {
+            size: 0,
+            query: {
+              match_all: {},
+            },
+          },
+        },
+      },
+    ],
+    triggers: [triggerBase],
+    ui_metadata: {
+      // Minimal metadata to help users identify this as a sample
+      description:
+        'Sample monitor created to help you connect a channel. Edit the action to point to your real endpoint before enabling.',
+    },
+  } as Monitor;
+}
+
+async function monitorExists(ctx: PluginTaskRunContext, name: string) {
+  try {
+    const payload = {
+      size: 0,
+      query: { term: { 'monitor.name.keyword': name } },
+    };
+    const { body } = await request(ctx, {
+      method: 'GET',
+      path: '/_plugins/_alerting/monitors/_search',
+      body: payload,
+    });
+    const total = body?.hits?.total?.value || 0;
+    return total > 0;
+  } catch (error) {
+    const _error = error as Error;
+    // If search fails (e.g., missing permissions/index), assume not exists
+    ctx.logger.warn(
+      `monitorExists error for [${name}]: ${_error?.message || _error}`,
+    );
+    return false;
+  }
+}
+
+async function ensureMonitor(
+  ctx: PluginTaskRunContext,
+  sample: SampleMonitorDef,
+  availableDefaultChannelIds?: Set<string>,
+) {
+  const { monitorName, channelId, message, severity } = sample;
+  const exists = await monitorExists(ctx, monitorName);
+  if (exists) {
+    ctx.logger.info(`Monitor already exists [${monitorName}]`);
+    return;
+  }
+
+  // Enforce: do not create sample monitors without their required channel
+  if (
+    !availableDefaultChannelIds ||
+    !availableDefaultChannelIds.has(channelId)
+  ) {
+    ctx.logger.info(
+      `Skipping sample monitor [${monitorName}] because required channel [${channelId}] is not present`,
+    );
+    return;
+  }
+
+  try {
+    await request(ctx, {
+      method: 'POST',
+      path: '/_plugins/_alerting/monitors',
+      querystring: { refresh: 'wait_for' },
+      body: buildMonitorBody(monitorName, severity, channelId, message),
+    });
+    ctx.logger.info(`Created sample monitor [${monitorName}]`);
+  } catch (error) {
+    const _error = error as Error;
+    // Don’t break health check; surface a clear warning
+    ctx.logger.warn(
```

Comment:
Why a warning instead of an error?
```suggestion
    ctx.logger.error(
```